docs/4.2/ssh_azuread.md

```diff
@@ -0,0 +1,204 @@
+  
+3. Select a Non-gallery application
+   ![Select Non-gallery application](img/azuread/azuread-3-selectnongalleryapp.png)
+   
+4. Enter the display name (Ex: Teleport)
+   ![Enter application name](img/azuread/azuread-4-enterappname.png)
+   
+5.Select properties under Manage and turn off User assignment required
+   ![Turn off user assignment](img/azuread/azuread-5-turnoffuserassign.png)
+   
+6. Select Single Sign-on under Manage and choose SAML
+   ![Select SAML](img/azuread/azuread-6-selectsaml.png)
+   
+7. Select to edit Basic SAML Configuration
+   ![Edit Basic SAML Configuration](img/azuread/azuread-7-editbasicsaml.png)
+   
+8. Put in the Entity ID and Reply URL the same proxy url https://teleport.example.com:3080/v1/webapi/saml/acs
+   ![Put in Entity ID and Reply URL](img/azuread/azuread-8-entityandreplyurl.png)
+   
+9. Edit User Attributes & Claims
+
+    i. Edit the Claim Name.  Change the name identifier format to Default. Make sure the source attribute is user.userprincipalname. 
+   ![Confirm Name Identifier](img/azuread/azuread-9a-nameidentifier.png)
+   
+    ii. Add a group Claim to have user security groups available to the connector
+   ![Put in Security group claim](img/azuread/azuread-9b-groupclaim.png)
+   
+    iii. Add a Claim to pass the username from transforming the AzureAD User name.
+   ![Add a transformed username](img/azuread/azuread-9c-usernameclaim.png)
+   
+   
+10. On the SAML Signing Certificate select to download SAML Download the Federation Metadata XML.  
+   ![Download Federation Metadata XML](img/azuread/azuread-10-fedmeatadataxml.png)
+
+!!! warning "Important"  This a important document.  Treat this file as you would a password.
+
+## Create a SAML Connector
+
+Now, create a SAML connector [resource](admin-guide.md#resources).  Replace the acs element with your Teleport address, update the group IDs with the actual AzureAD group ID values, and insert the downloaded Federation Metadata XML into the .
+Write down this template as `azure-connector.yaml`:
+
+```yaml
+kind: saml
+version: v2
+metadata:
+  # the name of the connector
+  name: azure-saml
+spec:
+  display: "Microsoft"
+  # acs is the Assertion Consumer Service URL. This should be the address of
+  # the Teleport proxy that your identity provider will communicate with.
+  acs: https://teleport.example.com:3080/v1/webapi/saml/acs
+  attributes_to_roles:
+    - {name: "http://schemas.microsoft.com/ws/2008/06/identity/claims/groups", value: "<group id 930210...>", roles: ["admin"]}
+    - {name: "http://schemas.microsoft.com/ws/2008/06/identity/claims/groups", value: "<group id 93b110...>", roles: ["dev"]}
+  entity_descriptor: |
+    <federationmedata.xml contents>
+```
+
+Create the connector using `tctl` tool:
+
+```bsh
+$ tctl create azure-connector.yaml
+```
+!!!Note Teleport will automatically transform the contents of the connector when viewed from the web UI.
```

Comment:
I think you have to format this as `!!! note` for `mkdocs` to pick it up and transform it.

You may also need a line break between it and the code block above.

Reply:
modified to a tip with the right format.